website/content/ja/contribute/projects/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: プロジェクト
+list_pages: true
+menu:
+  main:
+    weight: 20
+description: これは参加する機会のリストです
+---
+
+TAGプロジェクトは[GitHub Issues](https://github.com/cncf/tag-app-delivery/issues)で管理しています。現在あるプロジェクトや新しいプロジェクトを提案したい場合はこちらから始めてください。
+
+
+本項はissueにない現在進行中のコントリビューション候補です。このリストは完全なものではないことを留意してください。
+ここやGitHub Issusesにあなた向けのものがなくても大丈夫です!
```

Comment:
```suggestion
ここやGitHub Issusesにあなた向けのものがなくても大丈夫です！
```

If we follow the k8s l10n rules, just like with cncf/glossary, the "!" should be in full-width

FYI: https://kubernetes.io/ja/docs/contribute/localization/#basic-policy

> 記号類は感嘆符「！」と疑問符「？」のみ全角、それ以外は半角で表記